src/statement_generator.cpp

```diff
@@ -92,24 +92,43 @@ std::shared_ptr<GeneratorContext> StatementGenerator::GetDatabaseState(ClientCon
 	return result;
 }
 
+class RandomsConfig {
+public:
+	idx_t select_percentage = 60;
+	idx_t attach_percentage = 40;
+	idx_t attach_use_percentage = 50;
+	idx_t detach_percentage = 60;
+	idx_t set_percentage = 30;
+	idx_t delete_percentage = 40;
+
+	RandomsConfig(){
```

Comment:
Potentially another constructor with 
`RandomsConfig(string_t file_path)` so that if a file is passed, you can initialize the values to those in the file